### DISCUSSION TITLE
posts/getting-started-with-aws/

### DISCUSSION THREAD

**Discussion by giscus** [roninro/hugo-theme-puppet#5] [General]
# posts/getting-started-with-aws/

Confused and Puzzeled how and where to get started?
No, you are not alone. It is totally natural to get overwhalmed with so many services offered by AWS. Here, i aim to provide just enough information to get started. AWS, or Amazon Web Services does not need any introduction. It is market leader and 80% of cloud workload is on AWS.
Sometime, i wonder there should be a guide of simple steps that should guide me just enough to get started with my AWS account.

http://localhost:1313/posts/getting-started-with-aws/

**Comment by mumehta** [comment]
dsfs

**Comment by yeaheo** [comment]
1

**Reply by yeaheo** [reply]
1

**Reply by peng-du-cd** [reply]
peng.du

**Reply by ghost** [reply]
I like it

**Comment by ghost** [comment]
What is this?
